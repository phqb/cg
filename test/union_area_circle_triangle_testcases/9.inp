0 0 2
1 1.7320508076
1 -1.7320508076
4 0