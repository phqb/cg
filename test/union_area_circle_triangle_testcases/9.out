2.4567393972